server/app/services/export/QuestionJsonPresenter.java

```diff
@@ -201,6 +208,28 @@ public ImmutableMap<Path, Optional<String>> getJsonEntries(DateQuestion question
     }
   }
 
+  class EnumeratorJsonPresenter implements QuestionJsonPresenter<EnumeratorQuestion, String> {
+    @Override
+    public ImmutableMap<Path, Optional<String>> getJsonEntries(EnumeratorQuestion question) {
+      if (!question.isAnswered()) {
+        var path = question.getApplicantQuestion().getContextualizedPath();
```

Comment:
nitpick (non-blocking): since this line and line 219 are identical, you could remove this line of code by building the path before this `if` statement

Reply:
Ah thanks! Done.